pubsub/pulsar/pulsar.go

```diff
@@ -231,7 +231,7 @@ func (p *Pulsar) Publish(req *pubsub.PublishRequest) error {
 	if err != nil {
 		return err
 	}
-	if _, err = producer.Send(p.publishCtx, msg); err != nil {
+	if _, err = producer.Send(ctx, msg); err != nil {
```

Comment:
Here too please remove the now-unused `p.publishCtx` property from the struct

Reply:
Done.